### ISSUE TITLE
how to save and decode gray-jpeg images?

### ISSUE THREAD

**Issue by Nise-2-meet-U** [libffcv/ffcv#197]
My training data are one channle gray images.

**Comment by phelps-matthew** [comment]
Yes, it would be great if there were a class similar to `ffcv.fields.RGBImageField` for grayscale images that allows ones to write them with the same jpeg compression capability.

**Comment by GuillaumeLeclerc** [comment]
Hi!

There is currently a PR with this functionality. You can try this branch and give feedback in  #176 .
